S4TF_Tutorial_2.ipynb

```diff
@@ -396,31 +396,31 @@
         {
```

Comment:
Above this in the code, I'm not sure "truths" should be plural. "truth" might be a better name for that parameter.